### PR TITLE
Patch Panel Container

### LINKED ISSUE #89: Error message `z-index`
screen shot 2015-06-23 at 12 06 58

Screenshot says it all.

### PR DESCRIPTION
**Pull request by steelbrain**
Fixes #89 
see the bug for more info and comment in code for explanation.

### CODE REVIEW

**Comment by steelbrain** [comment]
@simurai shouldn't we be doing this in themes instead of patching via packages?

**Comment by iam4x** [comment]
I think you have to add another `z-index` on the top bar.

screen shot 2015-06-23 at 15 09 25

File: styles/linter-plus.less [hunk 1]
```diff
@@ -137,3 +137,9 @@ atom-text-editor::shadow .linter-highlight, .linter-highlight{
     color: @background-color-success;
   }
 }
+
+// The tooltip container's z-index is more than that of atom-panel-container
+// The tooltip overlays the panel if we don't do this
+atom-panel-container.bottom{
```
Comment (AsaAyers) [inline styles/linter-plus.less]: is `atom-panel-container` something of ours? From the name it doesn't sound like something we should mess with.

Comment (steelbrain) [inline styles/linter-plus.less]: I know, that's why it's called patching instead of fixing :smiley:

Comment (AsaAyers) [inline styles/linter-plus.less]: Can we adjust our tooltip's z-index instead?

Comment (steelbrain) [inline styles/linter-plus.less]: Nope, it hides behind `atom-text-editor` if we adjust the tooltip's z-index even a tiny bit

**Comment by steelbrain** [comment]
Closing, to be done by the themes. not us.

**Comment by simurai** [comment]
> Closing, to be done by the themes. not us.

Not sure if it's exactly the same issue, but might be worth to fix the z-index fighting in core, see: https://github.com/atom/atom/issues/6857

**Pull request closed without merging**